owasp-top10-2017-apps/a1/copy-n-paste/README.md

```diff
@@ -1,40 +1,166 @@
 # CopyNPaste API
-> This is a simple Golang API that contains an example of an Injection vulnerability.
 
 <img src="images/CopyNPaste.png" align="center"/>
 
-## What is Injection?
+CopyNPaste is Golang web application that uses an API and a simple front end to simulate a login page. It has both `/register` and `/login` routes that, by communicating with a MySQL database, enables users register and enter into a generic system.
```

Comment:
Typo: `is a Golang web application`and `enable users`

Reply:
Done: 43ce9ce